bin/goldrush

```diff
@@ -194,7 +197,15 @@ endif
 	@echo "Clean Done"
 
 # Set-up pipelines
-run: path-tigmint-ntLink check-G check-reads clean
+run: 
+	mkdir -p $(prefix)
+	cd $(prefix) && ln -sf ../$(reads).f* $(reads).fastq && goldrush run-in-dir reads=$(reads) G=$(G) t=$(t) z=$(z) track_time=$(track_time) k=$(k) w=$(w) tile=$(tile) b=$(b) u=$(u) a=$(a) o=$(o) x=$(x) h=$(h) s=$(s) m=$(m) M=$(M) r=$(r) P=$(P) d=$(d) span=$(span) dist=$(dist) k_ntLink=$(k_ntLink) w_ntLink=$(w_ntLink) rounds=$(rounds) polisher=$(polisher) polisher_mapper=$(polisher_mapper) shared_mem=$(shared_mem)
```

Comment:
Since we already have the $(long_reads) variable, could we use that instead of the wildcard?